src/models/Record.js

```diff
@@ -0,0 +1,20 @@
+const mongoose = require('mongoose');
+
+const { Schema, model } = mongoose;
+
+const RecordSchema = new Schema(
+  {
+    cover: { type: String, required: true, default: 'xyz' },
+    title: { type: String, required: true },
+    artist: { type: String, required: false },
+    year: { type: Number, required: false },
+  },
```

Comment:
Could we make these two required too, those are pretty important for a record, and it makes the work easier when we display it in the front-end. We are going to have something checking probably but in this case makes sense the artist and the year are required anyways, normally those are easy to find. In our case it will be fake of course, it just for consinstency.

Reply:
sure

Reply:
thanks